CHANGES.rst

```diff
@@ -22,6 +22,11 @@ Olivier Grisel
     thread is not the Python default. Thanks to Roman Yurchak for the
     suggestion.
 
+Karan Desai
+
+    py.test is used to run the tests instead of nosetests.
+    ``python setup.py test`` or ``python setup.py nosetests`` do not work
```

Comment:
Then maybe there are a few more places to modify in CHANGES.rst - I could notice single backticks in some lines..